### DISCUSSION TITLE
Can anyone explain how to scale in a few words?

### DISCUSSION THREAD

**Discussion by scaraliu** [FlowiseAI/Flowise#1718] [General]
I seem to have some issues on the number of users of the chat, and i run on a single machine.
I don't see in simple words the path to creating multiple machines, connected to a central mysql or other

**Comment by Jaredude** [comment]
What issues you are encountering with the number of users?

As far as multiple servers goes, if you setup 2+ servers that all have the same DATABASE_* env variables, they will all share the same flowise database and therefore the flows, tools, and chat history.

**Reply by scaraliu** [reply]
I thank you a lot for you answer, helped me a lot.
